forms-flow-web/src/components/ServiceFlow/details/UserSelectionDebounce.js

```diff
@@ -124,37 +122,38 @@ const UserSelectionDebounce = React.memo((props) => {
             formatOptionLabel={formatOptionLabel}
             onMenuClose={onClose}
             value={selectedValue}
+            data-testid="task-async-user-select"
           />
         </Col>
         <Col sm={2} className="p-0 no-padding-left">
-          <DropdownButton
-            id="dropdown-basic-button"
-            title={<i className="fa fa-filter" />}
-            size="sm"
-            variant="secondary"
-          >
-            {UserSearchFilterTypes.map((UserSearchFilterType, idx) => {
-              return (
-                <Dropdown.Item
-                  key={idx}
-                  className="click-element"
-                  onClick={() => setSearchTypeOption(UserSearchFilterType)}
-                >
-                  <Form.Check
-                    type="radio"
-                    id={UserSearchFilterType.searchType}
-                    key={UserSearchFilterType.searchType}
-                    label={UserSearchFilterType.title}
-                    value={UserSearchFilterType.searchType}
-                    checked={
-                      searchTypeOption.searchType ===
-                      UserSearchFilterType.searchType
-                    }
-                  />
-                </Dropdown.Item>
-              );
-            })}
-          </DropdownButton>
+          <Dropdown>
+            <Dropdown.Toggle variant="secondary" id="dropdown-basic" data-testid="assignee-search-filter-dropdown-toggle">
+              <i className="fa fa-filter" />
+            </Dropdown.Toggle>
+            <Dropdown.Menu className="searchtype-dropdown">
+              {UserSearchFilterTypes.map((UserSearchFilterType, idx) => (
+                <div key={idx} className="mb-2 mx-2" data-testid={`assignee-search-filter-option-${idx}`}>
+                  <label className="form-check-label">
+                    <input
+                      className="form-check-input me-2"
+                      type="radio"
+                      id={UserSearchFilterType.searchType}
+                      name="searchType"
+                      value={UserSearchFilterType.searchType}
+                      onChange={() => setSearchTypeOption(UserSearchFilterType)}
+                      checked={
+                        searchTypeOption.searchType ===
+                        UserSearchFilterType.searchType
+                      }
+                      data-testid={`assignee-filter-radio-${UserSearchFilterType.searchType}`}
```

Comment:
use filter-option instead of filter-radio

Reply:
changed